test/functionalTest/cases/1306_string2double/string2double_not_a_number.test

```diff
@@ -148,12 +148,12 @@ echo
 01. Not a number for q, op >
 ============================
 HTTP/1.1 400 Bad Request
-Content-Length: 63
+Content-Length: 103
 Content-Type: application/json
 Date: REGEX(.*)
 
 {
-    "description": "invalid query expression",
+    "description": "values of type /String/ not supported for operator /GreaterThan/",
```

Comment:
This is going to change in the future (see https://github.com/telefonicaid/fiware-orion/issues/1945).

NTC (just to prepare for the future :)